book/page_creation.rst

```diff
@@ -873,234 +548,43 @@ format you prefer:
 
         // ...
 
-.. note::
-
-   You'll learn exactly how to load each file/format in the next section
-   `Environments`_.
-
-Each top-level entry like ``framework`` or ``twig`` defines the configuration
-for a particular bundle. For example, the ``framework`` key defines the configuration
-for the core Symfony FrameworkBundle and includes configuration for the
-routing, templating, and other core systems.
-
-For now, don't worry about the specific configuration options in each section.
-The configuration file ships with sensible defaults. As you read more and
-explore each part of Symfony, you'll learn about the specific configuration
-options of each feature.
-
-.. sidebar:: Configuration Formats
-
-    Throughout the chapters, all configuration examples will be shown in all
-    three formats (YAML, XML and PHP). Each has its own advantages and
-    disadvantages. The choice of which to use is up to you:
-
-    * *YAML*: Simple, clean and readable (learn more about YAML in
-      ":doc:`/components/yaml/yaml_format`");
-
-    * *XML*: More powerful than YAML at times and supports IDE autocompletion;
-
-    * *PHP*: Very powerful but less readable than standard configuration formats.
-
-Default Configuration Dump
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You can dump the default configuration for a bundle in YAML to the console using
-the ``config:dump-reference`` command. Here is an example of dumping the default
-FrameworkBundle configuration:
-
-.. code-block:: bash
-
-    $ app/console config:dump-reference FrameworkBundle
+The ``framework`` key configures FrameworkBundle, the ``twig`` key configures
+TwigBundle and so on. A *lot* of behavior in Symfony can be controlled.
```

Comment:
I don't know if this phrase is entirelly correct: `A *lot* of behavior in Symfony can be controlled.`

In fact, "everything" can be controlled in Symfony. You may want to say that "a lot of behavior can be controlled just by changing one option in this configuration file".